Clients/Xamarin.Interactive.Client.Web/Controllers/GistController.cs

```diff
@@ -0,0 +1,42 @@
+//
```

Comment:
This entire class needs formatting per our .editorconfig.

Reply:
Weird, I have the EditorConfig plugin for VSCode...